tests/dem/insertion_non_uniform.cc

```diff
@@ -67,6 +67,9 @@ test()
   dem_parameters.insertion_info.inserted_this_step                   = 10;
   dem_parameters.insertion_info.distance_threshold                   = 2;
   dem_parameters.lagrangian_physical_properties.particle_type_number = 1;
+  dem_parameters.lagrangian_physical_properties.size_distribution_type =
+    Parameters::Lagrangian::LagrangianPhysicalProperties::
+      size_distribution_type::uniform;
```

Comment:
I am not familiar with this tests but why would you have two tests with uniform distribution? Isn't one for non-uniform and one for uniform? At least this is what I get from the name of the tests...

Reply:
Are you talking about the name of the file? 

Previously (less than a month ago), we had three types of INSERTION methods called uniform, non-uniform and plane. The uniform insertion was not useful since you could obtained the same type of insertion location with the non-uniform one. The uniform insertion method got deprecated and the non-uniform insertion method got renamed volume insertion method. In brief, we could technically rename this test `insertion_volume.cc` and deprecated the `insertion_uniform.cc` 

Now, this PR is about size distribution, so the diameter value at which particles are being inserted. Currently we have two types of size distribution, uniform (every particle with the same diameter) and normal (particle diameters follow a normal distribution). 

Like I said in a previous comment, the difference between uniform and normal size distribution is the standard deviation (equal or not equal to zero). If you check the parsing function in `core/parameter_lagrangian.cc`, line 70, you will see that the `enum class size_distribution_type ` was previously never set to uniform or normal, but now it needs to be and this is why you see this modification in this test. 

I hope this helps.  :-)

Reply:
I see... in that case I do think that it would be better to rename the files? So that the name of the test actually reflects what is being tested...

Reply:
I agree with @lpsaavedra

Reply:
Ok, will renamed the test. What should I do we the insertion_uniform.cc test? deprecate or renamed to insertion_volume_1.cc and insertion_volume_2.cc ?

Reply:
Yeah

Reply:
This was not a yes or no question, but I will assume you said yes to the second option. 🫠